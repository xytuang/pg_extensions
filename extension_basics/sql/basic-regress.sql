CREATE EXTENSION basic;

SELECT * FROM BASIC_TABLE;

SELECT add(2,3);
