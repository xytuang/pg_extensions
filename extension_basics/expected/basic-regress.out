CREATE EXTENSION basic;
SELECT * FROM BASIC_TABLE;
 id | name 
----+------
(0 rows)

SELECT add(2,3);
 add 
-----
   5
(1 row)

